dev-tools/audius-compose

```diff
@@ -310,6 +311,7 @@ def load_env(protocol_dir, service, environment):
 @click.option(
     "--no-solana", is_flag=True, help="Don't bring up solana-test-validator container"
 )
+@click.option("--no-graphql", is_flag=True, help="Don't bring up graphql container")
```

Comment:
i know this kind of goes against the established pattern but shouldn't the default here be to **not** start up the gql container? that way we have to explicitly bring it up until it's ready for release, that also would avoid the need for other to pass this flag just to run their local stack